src/carbonable/erc3525/library.cairo

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+// Carbonable Contracts for Cairo v0.0.1 (carbonable/erc3525/library.cairo)
+
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.uint256 import Uint256
+
+//
+// Events
+//
+
+@event
+func TransferValue(_fromTokenId: Uint256, _toTokenId: Uint256, _value: Uint256) {
+}
+
+@event
+func ApprovalValue(_tokenId: Uint256, _operator, _value: Uint256) {
+}
+
+@event
+func SlotChanged(_tokenId: Uint256, _oldSlot: Uint256, _newSlot: Uint256) {
+}
+
```

Comment:
I think it's missing an event, from the `IERC3525SlotApprovable` specification from [eip-3525](https://eips.ethereum.org/EIPS/eip-3525). 

The event `event ApprovalForSlot(address indexed _owner, uint256 indexed _slot, address indexed _operator, bool _approved);`

Reply:
I plan to put this event in another file in a `presets` folder since slot level approval is optional.